src/layouts/flex/lv_flex.c

```diff
@@ -116,7 +116,8 @@ void lv_obj_set_flex_align(lv_obj_t * obj, lv_flex_align_t main_place, lv_flex_a
 void lv_obj_set_flex_grow(lv_obj_t * obj, uint8_t grow)
 {
     lv_obj_set_style_flex_grow(obj, grow, 0);
-    lv_obj_mark_layout_as_dirty(lv_obj_get_parent(obj));
+    lv_obj_t * parent = lv_obj_get_parent(obj);
+    if(parent) lv_obj_mark_layout_as_dirty(parent);
```

Comment:
Can you add a test case to cover this scenario? The coverage report does not cover the entire branch:
![3195938f-2073-4d8a-a56d-0766f8d91b94](https://github.com/user-attachments/assets/1e49f845-a8c0-4246-8bae-87d4bd1932a6)